package.json

```diff
@@ -1,30 +1,31 @@
 {
   "name": "opds-feed-parser",
-  "version": "0.0.16",
+  "version": "0.0.17",
   "description": "OPDS feed parser",
   "author": "NYPL",
   "repository": {
     "type": "git",
     "url": "git@github.com:NYPL-Simplified/opds-feed-parser.git"
   },
-  "main": "lib/index.js",
-  "typings": "lib/index.d.ts",
+  "main": "lib/src/index.js",
+  "typings": "lib/test/index.d.ts",
   "license": "Apache-2.0",
   "scripts": {
-    "prepublish": "rm -rf lib; typings install; tsc",
-    "test": "tslint -c tslint.json src/*.ts test/*.ts && mocha test-build/test/*.js",
-    "pretest": "tsc test/*.ts typings/main/ambient/**/*.ts --outDir test-build --module commonjs"
+    "prepublish": "rm -rf lib; tsc",
+    "lint": "tslint -c tslint.json src/*.ts test/*.ts",
+    "test": "rm -rf lib && npm run lint && tsc && mocha lib/test/*.js"
```

Comment:
This makes it easier to compile all together for the tests. I kept getting errors of type configuration/files not being read when using the command line for a different output directory.

Reply:
Does this also mean tests will be in the published package though? That's not ideal.

Reply:
I added it in the npmignore file and that should be enough.

Reply:
Just checked when importing in the opds-web-client and the compiled tests folder is ignored from the published packaged.